host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1_cuda.cmake

```diff
@@ -0,0 +1,168 @@
+#------------------------------------------------------------------------------
```

Comment:
Ah, because `_cuda`. I added the "special case" section in the naming host config section of the spack package. I think I still need to remove the other lassen host config.